src/main/java/net/jodah/failsafe/SyncFailsafe.java

```diff
@@ -114,6 +115,27 @@ public void run(ContextualRunnable runnable) {
   }
 
   /**
+   * Creates a proxy instance of the provided {@link T} instance such that all public methods
+   * use the {@link RetryPolicy} and {@link CircuitBreaker} set.
+   * The returned proxy adheres to the interface provided except for two cases:
+   * <ol>
+   *     <li>CircuitBreakerException: The methods may throw this type of
+   *     exception if the circuit breaker is open.</li>
+   *     <li>UndeclaredThrowableException: If you provide a fallback which throws
+   *     an exception undeclared by the interface.</li>
```

Comment:
Minor wording improvement: _which throws an exception **not declared** by the interface_